dev/Deployment/DeploymentManager.cpp

```diff
@@ -164,6 +165,14 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentInitializeOptions const& deploymentInitializeOptions,
         bool isRepair)
     {
+        if (isInitializedByDefault)
+        {
+            HRESULT hr = HRESULT_FROM_WIN32(E_UNEXPECTED);
+            winrt::hstring message = L"DeploymentManager can only be initialized once and has already been initialized by default. "
+                L"If you would like to explicitly initialize, add <WindowsAppSdkDeploymentManagerInitialize>false</WindowsAppSdkDeploymentManagerInitialize> to your .csproj file.";
```

Comment:
The error could be from either source - so maybe a more neutral suggestion
```suggestion
                L"Please either remove your explicit initialization call, or disable auto-initialization by adding <WindowsAppSdkDeploymentManagerInitialize>false</WindowsAppSdkDeploymentManagerInitialize> to your .csproj file.";
```